distribution/docs/src/main/resources/_contents/_developing/developing-intro-contents.adoc

```diff
@@ -0,0 +1,823 @@
+* Implements a dynamic component model that allows application updates without downtime.
+Components can be added or updated in a running system.
+* Standardized Application Configuration (ConfigurationAdmin and MetaType)
+
+More information on OSGi is available at http://www.osgi.org/[OSGI].
+
+==== Built on Apache Karaf
+
+Apache Karaf is a FOSS product that includes an OSGi framework and adds extra functionality, including:
+
+${command-console}:: Provides command line administration of the OSGi container. All functionality in the ${admin-console} can also be performed via this command line console.
+Logging:: Provides centralized logging to `data/log/${branding-lowercase}.log`. Security logging is provided at `data/log/security.log`. Ingest error logging can be viewed in `data/log/ingest_error.log`.
+Provisioning:: Of libraries or applications.
+Security:: Provides a security framework based on Java Authentication and Authorization Service (JAAS).
+Deployer:: Provides hot deployment of new bundles by dropping them into the `<INSTALL_DIR>/deploy` directory.
+Blueprint:: Provides an implementation of the OSGi Blueprint Container specification that defines a dependency injection framework for dealing with dynamic configuration of OSGi services.
+** ${branding} uses the Apache Aries implementation of Blueprint.
+More information can be found at http://aries.apache.org/modules/blueprint.htm[Blueprint].
+Spring DM:: An alternative dependency injection framework.
+${branding} is not dependent on specific dependency injection framework, but Blueprint is recommended.
+
+==== Additional Upstream Dependencies
+
+${branding} is a customized distribution of Apache Karaf, and therefore includes all the capabilities of Apache Karaf.
+${branding} also includes additional FOSS components to provide a richer set of capabilities.
+Integrated components include their own dependencies, but at the platform level, ${branding} includes the following upstream dependencies:
+
+Apache CXF:: Apache CXF is an open source services framework. CXF helps build and develop services using front end programming APIs, such as JAX-WS and JAX-RS. More information can be found at http://cxf.apache.org.
+Apache Commons:: Provides a set of reusable Java components that extends functionality beyond that provided by the standard JDK  (More info available at http://commons.apache.org)
+OSGeo GeoTools:: Provides spatial object model and fundamental geometric functions, which are used by ${branding} spatial criteria searches. More information can be found at http://geotools.org/.
+
+For a full list of dependencies, refer to the <<_dependency_list,Dependency List>>.
+
+=== OSGi Services
+
+Services consist of:
+
+An API Bundle:: The API, written as Java interfaces, defines the contract of the service and should, to the extent possible, reference only those concrete classes that are loaded by the root classloader.
```

Comment:
How about "that are loaded by the root classloader (i.e., the classes in the `java.*` package)."?